alerts/plugin.yaml

```diff
@@ -51,7 +51,7 @@ spec:
       required: false
       type: list
     - name: alerts.alertmanager.serviceMonitor.additionalLabels
-      description: Additional ServiceMonitor labels. Name of the kube-monitoring PluginConfig to monitor Alertmanager.
+      description: Additional ServiceMonitor labels. Name of the kube-monitoring Plugin to monitor Alertmanager.
       default:
         pluginconfig: kube-monitoring
```

Comment:
Same here: Plugin owner would need to know, if label changes?

Reply:
This only changed the description, but not the label itself. I will revert the description change and leave the defaults/ labels as `pluginconfig: kube-monitoring`, as this is something only used internally in this plugin and can be adjusted later. @richardtief FYI, I believe this can be done after changing the CRDs as these labels are only used internally in the alerts & kube-monitoring plugins, right?

Reply:
Yes, they simply label the resources provided with the plugin. Can be adjusted later